lib/Service/Note.php

```diff
@@ -60,12 +60,12 @@ public function getExcerpt(int $maxlen = 100) : string {
 		if (!empty($title)) {
 			$length = strlen($title);
```

Comment:
We have to switch from `strlen` to `mb_strlen`, otherwise we will have problems if the title contains multi-byte characters.

Reply:
Sounds obvious. Thanks for pointing out!